website/content/docs/configure.mdx

```diff
@@ -13,69 +13,42 @@ so you generally don't have to worry about it until you want to tweak a
-`PACKER_HOME_DIR`.
-
-## Packer's config file
-
-Packer can optionally read a JSON file for the end user to set core settings.
-The config file of Packer will be looked up on the following paths:
-
-| Unix                             | Windows                          |
-| -------------------------------- | -------------------------------- |
-| `${PACKER_CONFIG}`               | `%PACKER_CONFIG%`                |
-| `PACKER_HOME_DIR/.packerconfig`  | `PACKER_HOME_DIR/packer.config/` |
-| `${XDG_CONFIG_HOME}/packer`      |                                  |
-| `PACKER_HOME_DIR/.config/packer` |                                  |
-
 ## Packer's config directory
 
 Packer's configuration directory can potentially contain plugins and internal
-Packer files. The config dir of Packer will be looked up on the following paths:
+Packer files. The Packer config directory will be looked up on the following paths:
 
 | Unix                        | Windows                     |
 | --------------------------- | --------------------------- |
-| `PACKER_HOME_DIR/.packer.d` | `PACKER_HOME_DIR/packer.d/` |
+| `${HOME}/.config/packer/`   | `%APPDATA%\packer.d\`       |
+
+-> **Note:** On Unix systems, Packer defaults to using the XDG base directory specification. 
+When the environment variable `PACKER_CONFIG_DIR` is unset or empty a default equal to `$HOME/.config/packer` should be used. 
+In all other cases, where there is an existing older style `.packer.d` directory (e.g `$HOME/.packer.d/`)  or PACKER_CONFIG_DIR is not empty
+the older configuration directory will be used. 
 
 Examples:
 
-- On a Unix system, if the `$PACKER_CONFIG_DIR` env var is set to
+- On a Unix system, if the `$PACKER_CONFIG_DIR` environment variable is set to
   `/home/packer`, the config directory will be: `/home/packer/.packer.d/` and
   other values will not be checked.
-- On a Unix system, if the `HOME` env var is `/home/azr` or the `USER` env var
-  is `azr`, then the config directory will default to `/home/azr/.packer.d/`.
-- On a Windows system, if the `PACKER_CONFIG_DIR` env var is set to `C:/`,the
+- On a Windows system, if the `PACKER_CONFIG_DIR` environment variable is set to `C:/`,the
   config directory will be: `C:/packer.d/` and other values will not be checked.
 
-## Packer's plugin directory
-
-@include "plugins/plugin-location.mdx"
+<a id="packer-s-config-file"></a>
+## Packer's config file (deprecated)
 
-The format of the configuration file is basic JSON.
+Packer can optionally read a JSON file for the end user to set core settings.
+The config file of Packer will be looked up on the following paths:
 
-## Packer's cache directory
+| Unix                             | Windows                           |
+| -------------------------------- | --------------------------------- |
+| `${PACKER_CONFIG}`               | `%PACKER_CONFIG%`                 |
+| `PACKER_CONFIG_DIR/.packerconfig`| `PACKER_CONFIG_DIR\packer.config\`|
```

Comment:
This might need a little rework in the code. Looking at the logs I see 
```
023/07/05 15:00:50 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2023/07/05 15:00:50 [INFO] PACKER_CONFIG env var set; attempting to open config file: /Users/dev/.packerconfig
```